src/graphql/resolvers/queries/partners.js

```diff
@@ -1,23 +1,24 @@
+/* eslint-disable import/prefer-default-export */
+import debug from 'debug';
 import partnerStore from '../../../dataSources/cloudFirestore/partner';
 
-const resolvers = {
-  partner: async (parent, { id }, { dataSources }) =>
-    partnerStore(dataSources.firestore).get(id),
+const dlog = debug('asdf');
 
-  partnerBySlug: async (parent, { slug }, { dataSources }) =>
-    partnerStore(dataSources.firestore).findBySlug(slug),
-
-  partners: async (parent, args, { dataSources }) =>
-    partnerStore(dataSources.firestore).getAll(),
-};
+export const fieldResolvers = {
+  PartnersQuery: {
+    all: async (parent, args, { dataSources }) => {
+      dlog('asdf');
+      return partnerStore(dataSources.firestore).getAll();
+    },
 
-export default resolvers;
+    partner: async (parent, { id }, { dataSources }) => {
+      dlog('asdf');
+      return partnerStore(dataSources.firestore).get(id);
+    },
 
-export const refResolvers = {
-  Partner: {
-    // eslint-disable-next-line no-underscore-dangle
-    __resolveReference(partner, { dataSources }) {
-      return partnerStore(dataSources.firestore).get(partner.id);
+    partnerBySlug: async (parent, { slug }, { dataSources }) => {
+      dlog('asdf');
```

Comment:
you need that log?

Reply:
yea that's debug statements.. you will start to see a lot more of them

Reply:
But it could say something meaningful 🤪